examples/3d/pbr_debug.rs

```diff
@@ -0,0 +1,103 @@
+//! Loads and renders a glTF file as a scene and displays rendering debug visualizations.
+
+use bevy::{pbr::PbrDebug, prelude::*};
+
+fn main() {
+    App::new()
+        .insert_resource(AmbientLight {
+            color: Color::WHITE,
+            brightness: 1.0 / 5.0f32,
+        })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(animate_light_direction)
+        .add_system(cycle_pbr_debug)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn_bundle(Camera3dBundle {
+        transform: Transform::from_xyz(0.7, 0.7, 1.0).looking_at(Vec3::new(0.0, 0.3, 0.0), Vec3::Y),
+        ..default()
+    });
+    const HALF_SIZE: f32 = 1.0;
+    commands.spawn_bundle(DirectionalLightBundle {
+        directional_light: DirectionalLight {
+            shadow_projection: OrthographicProjection {
+                left: -HALF_SIZE,
+                right: HALF_SIZE,
+                bottom: -HALF_SIZE,
+                top: HALF_SIZE,
+                near: -10.0 * HALF_SIZE,
+                far: 10.0 * HALF_SIZE,
+                ..default()
+            },
+            shadows_enabled: true,
+            ..default()
+        },
+        ..default()
+    });
+    commands.spawn_bundle(SceneBundle {
+        scene: asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0"),
+        ..default()
+    });
+}
+
+fn cycle_pbr_debug(
+    time: Res<Time>,
+    mut next_switch: Local<Option<f64>>,
+    mut pbr_debug: ResMut<PbrDebug>,
+) {
+    let now = time.seconds_since_startup();
+    if next_switch.is_none() {
+        *next_switch = Some(now + 2.0);
+    }
+    if now > next_switch.unwrap() {
```

Comment:
I think this should use a keyboard input instead of a timer. Probably juste using space would be fine. Or maybe just use V to be consistent with the scene_viewer

Reply:
I'd be fine with that.